tests/unit/processors/processVrmRequest.test.ts

```diff
@@ -0,0 +1,51 @@
+import { APIGatewayProxyEvent } from 'aws-lambda';
+import {
+  TechrecordGet,
+} from '../../../src/models/post';
+import { processPatchVrmRequest } from '../../../src/processors/processVrmRequest';
+import postCarData from '../../resources/techRecordCarPost.json';
+
+describe('processVrmRequest', () => {
+  it('returns updated records to archive and to add', () => {
+    const event = {
```

Comment:
My advice would be you don't actually need the whole event for the unit test (even if you don't refactor). I know other people have done this too but you can spoof it easily for a test. look at the get handler tests, just have an object with the needed info, then cast it to unknown then as APIGatewayProxyEvent and it should work. For example all the headers, things like method, resource path, the code does not care they're just data from APIG. Again appreciate others have done this, but I will get down to removing them too.